nimbus-features/onboardingFrameworkFeature.yaml

```
@@ -30,7 +30,7 @@ features:
             ALWAYS:                   "true"
             NEVER:                    "false"
           cards:
-            - name: "welcome"
+            wecome:
```

Typo? I guess should be welcome
```suggestion
            welcome:
```

---

OOPS! Applying this, but not re-running BR for it.